autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/pull_over/pull_over.param.yaml

```
@@ -24,8 +24,8 @@
       use_occupancy_grid: true
       use_occupancy_grid_for_longitudinal_margin: false
       occupancy_grid_collision_check_margin: 0.0
-      theta_size: 360
-      obstacle_threshold: 60
+      theta_size: 360.0
+      obstacle_threshold: 60.0
```

fix for https://github.com/autowarefoundation/autoware.universe/pull/3048/commits/3e00b9a1b366a60e033fea9b6ab66eaddcf415a7
```suggestion
      theta_size: 360
      obstacle_threshold: 60
```